vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -225,39 +249,38 @@ An overview of the current connection and profile information is provided via `e
 ```
 ent pro new [profileName] [EntandoAppName] [namespace]
 
-This sets minimal profile data and explains next steps.
+This sets the minimal profile data and explains next steps.
 ```
 
 2. Link the current profile to a Kubernetes context (alias of `ent attach-kubectx`)
 ```
 ent pro link [contextName]
 ```
-This instructs `ent` how to connect to the Kube containing the EntandoApp.
+This instructs ent how to connect to the Kubernetes containing the Entando Application.
```

Comment:
Again, I think ent looks lonely and tiny without the backticks. I think this ent and those below need the backticks

Reply:
lol :) i considered using ENT when i was going through the release notes but that's a bit much and not consistent with how it's used in commands (or referenced as "ent" on many pages...)